### DISCUSSION TITLE
Welcome to Choria Discussions!

### DISCUSSION THREAD

**Discussion by ripienaar** [choria-io/general#1] [General]
## Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

We can also be found on [slack](https://puppetcommunity.slack.com/) in the `#choria` channel.

## Links

 * Choria [Home](https://choria.io) | [Docs](https://choria.io/docs) | [Blog](https://choria.io/blog)
 * [Code of Conduct](https://github.com/choria-io/.github/blob/master/CODE_OF_CONDUCT.md)
 * [Contribution Guidelines](https://github.com/choria-io/.github/blob/master/CONTRIBUTING.md)
 * [Security Policy](https://github.com/choria-io/.github/blob/master/SECURITY.md)

**Comment by vjanelle** [comment]
Vincent Janelle, and I had opinions about the TLS code quite often.  Those opinions are now merged.

**Reply by ripienaar** [reply]
Wish you also had merged opinions about TLS verification on go 1.15 :)
